spec/features/saml_spec.rb

```diff
@@ -4,8 +4,6 @@
 
   it { expect { visit '/users/sign_in' }.to_not raise_error }
 
-  after { cleanup_jetty }
```

Comment:
This entire feature is based around User model (aka ActiveRecord/Mysql). It has nothing to do with Fedora. Why are constantly cleaning fedora in this spec?